extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -154,7 +154,8 @@ export class AmpAnalytics extends AMP.BaseElement {
     // Trigger callback can be synchronous. Do the registration at the end.
     for (const k in this.config_['triggers']) {
       if (this.config_['triggers'].hasOwnProperty(k)) {
-        const trigger = this.config_['triggers'][k];
+        let trigger = null;
```

Comment:
this is unfortunate but NTI doesn't like `const` here for some reason (either a bug or transpiler is creating some ES5 code where `trigger` is not a real const and the symbol is reused because in the loop). I have seen only a few cases like this, so something we get get by for now.